package.json

```diff
@@ -10,25 +10,31 @@
     "gatsby-plugin-manifest": "^2.6.1",
     "gatsby-plugin-offline": "^3.4.0",
     "gatsby-plugin-react-helmet": "^3.4.0",
-    "gatsby-plugin-sharp": "^2.8.0",
     "gatsby-plugin-s3": "^0.3.8",
+    "gatsby-plugin-sharp": "^2.8.0",
     "gatsby-source-filesystem": "^2.5.0",
     "gatsby-transformer-sharp": "^2.6.0",
-    "prop-types": "^15.7.2",
     "mirador": "^3.0.0",
     "mirador-image-tools": "^0.10.0",
-    "typy": "^3.1.0",
+    "prop-types": "^15.7.2",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "react-helmet": "^6.1.0"
+    "react-helmet": "^6.1.0",
+    "typy": "^3.1.0",
+    "@loadable/component": "^5.14.1"  
```

Comment:
Just wanted to call this line out @danthewolfe.  I use this library to not load mirador in test.

Reply:
👍  I could not get this to work right in MARBLE when I tried using it before. I am glad it works in this scenario.

Reply:
nm I was mistaken this does not work

Reply:
replaced with a mock which does work.

Reply:
👍